include/core/Basis.hpp

```diff
@@ -307,7 +307,7 @@ class Basis {
 
 	Basis();
 
-	const Vector3 &operator[](int axis) const {
```

Comment:
How did that compile in the first place?

Reply:
Unfortunately, returning a reference to a temporary is only a warning not an error.